Code/Framework/AzCore/CMakeLists.txt

```diff
@@ -20,6 +20,20 @@ if(PAL_TRAIT_PROF_PIX_SUPPORTED)
     endif()
 endif()
 
+ly_add_target(
+    NAME O3DEKernel ${PAL_TRAIT_MONOLITHIC_DRIVEN_LIBRARY_TYPE}
+    NAMESPACE AZ
+    FILES_CMAKE
+        AzCore/o3dekernel_files.cmake
+    PLATFORM_INCLUDE_FILES
+        ${pal_dir}/o3dekernel_${PAL_PLATFORM_NAME_LOWERCASE}.cmake
+    INCLUDE_DIRECTORIES
+        PUBLIC
+            .
+            ${pal_dir}
+            ${common_dir}
+)
+
```

Comment:
Are there plans to be moving the O3DEKernel library to it's own folder structure?